schemas/HPXMLDataTypes.xsd

```diff
@@ -1845,10 +1845,29 @@
 	<xs:simpleType name="ProgramCertificate">
 		<xs:restriction base="xs:string">
 			<xs:enumeration value="Home Performance with Energy Star"/>
+			<xs:enumeration value="Energy Star Certified"/>
+			<xs:enumeration value="DOE Zero Energy Ready Home"/>
 			<xs:enumeration value="LEED Certified"/>
 			<xs:enumeration value="LEED Silver"/>
 			<xs:enumeration value="LEED Gold"/>
 			<xs:enumeration value="LEED Platinum"/>
+			<xs:enumeration value="Indoor airPLUS"/>
+			<xs:enumeration value="WaterSense"/>
+			<xs:enumeration value="NGBS Single-Family Bronze"/>
+			<xs:enumeration value="NGBS Single-Family Silver"/>
+			<xs:enumeration value="NGBS Single-Family Gold"/>
+			<xs:enumeration value="NGBS Single-Family Emerald"/>
+			<xs:enumeration value="NGBS Multifamily Bronze"/>
+			<xs:enumeration value="NGBS Multifamily Silver"/>
+			<xs:enumeration value="NGBS Multifamily Gold"/>
+			<xs:enumeration value="NGBS Multifamily Emerald"/>
+			<xs:enumeration value="NGBS Remodeling Bronze"/>
+			<xs:enumeration value="NGBS Remodeling Silver"/>
+			<xs:enumeration value="NGBS Remodeling Gold"/>
+			<xs:enumeration value="NGBS Remodeling Emerald"/>
+			<xs:enumeration value="Enterprise Green Communities"/>
+			<xs:enumeration value="Earthcraft"/>
+			<xs:enumeration value="Environments for Living"/>
 			<xs:enumeration value="other"/>
```

Comment:
I added gold, silver, bronze, emerald to the appropriate NGBS certifications and removed the land development one because it doesn't seem to be relevant.